Xamarin.Forms.Platform.Android/CollectionView/CarouselViewRenderer.cs

```diff
@@ -131,6 +134,15 @@ protected override void UpdateItemSpacing()
 			base.UpdateItemSpacing();
 		}
 
+		void InitializeSnapBehavior()
```

Comment:
Shouldn't this be done on Core ?

Reply:
Mnn, We already set the **SnapPointsType** property in the constructor https://github.com/xamarin/Xamarin.Forms/blob/d7a9c85774502b98460fc740068e264f332bfd82/Xamarin.Forms.Core/Items/CarouselView.cs#L164. The problem comes if you use a custom ItemsLayout like:
```
<CarouselView 
     Position="0"
     CurrentItem="{Binding CurrentItem, Mode=TwoWay}"
     ItemsSource="{Binding Items}"
     ItemTemplate="{StaticResource ItemTemplate}">
     <CarouselView.ItemsLayout>
          <ListItemsLayout
               Orientation="Horizontal"/>
     </CarouselView.ItemsLayout>
</CarouselView>
```

At the end, in this case the SnapPointsType property is not being established and the default value is **None**. In this case the CarouselView will not span until set the property. So, we can do nothing; or in case of having the property value equals to None (as in the previous example) use MandatorySingle (the value assigned in the constructor). In this PR we take the second path, but what do you think?.